samza-core/src/test/java/org/apache/samza/operators/impl/TestOperatorImplGraph.java

```diff
@@ -320,7 +320,7 @@ public void testOperatorGraphInitAndClose() {
   }
 
   /**
-   * Creates an identity map function that appends to the provided lists when init/close is invoked.
+   * Creates an identity map function that appends to the provided lists when createStream/close is invoked.
```

Comment:
Why changing this file? Seems irrelevant?

Reply:
Inadvertent refactor from IntelliJ. Fixed. Thanks!